packages/sdk-bridge/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### 1.0.0-rc.19
 
 - fix: numeric fault overflow when supporting infinite token approval
+- feat: try send/sendNative and process calls using callStatic
```

Comment:
shouldn't this be in unreleased?

Reply:
That package was about to be released but I asked to get this in at the same time